### ISSUE TITLE
[a11y]: SR announces verbose information on any updates to DataTable

### ISSUE THREAD

**Issue by preranagiri22** [carbon-design-system/carbon#19236]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.80.1

### Automated testing tool and ruleset

JAWS

### Assistive technology

JAWS

### Description

When any updates are made to the DataTable screen reader announces entire table information. The announcement of verbose information can negatively impact the overall user experience, leading to feelings of inefficiency, confusion, and decreased productivity.
 

![Image](https://github.com/user-attachments/assets/cb2f63a1-f603-44f7-b1cd-5edcc517aa3f)

### WCAG 2.1 Violation

1.3.1 Info and Relationships

### Reproduction/example

NA

### Steps to reproduce

NA

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Hi, this issue has been moved to the Carbon Charts repository for better tracking.
